roulette/package.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
As was mentioned in the previous PR: please get rid of the redundant folder nesting.
In your repository "netflix-react" at the root level (i.e. at the level of `netflix-react/`) there should be:
- only 1 pakcage.json
- only 1 pakcage-lock.json
- only 1 .eslintrc, 1 .gitignore, 1 README.md, etc
There's no need to keep a separate copy of these files under several unnecessary folders.

Move the contents of `netflix-react/roulette/*` to `netflix-react/`.
Move the contents of `netflix-react/rm1/src/components/*` to `netflix-react/misc`
Remove rm1, rm2 and roulette folders completely.